ui/app/components/app/alerts/unconnected-account-alert/unconnected-account-alert.js

```diff
@@ -35,7 +38,7 @@ const SwitchToUnconnectedAccountAlert = () => {
     <Popover
       contentClassName="unconnected-account-alert__content"
       title={t('notConnected')}
-      subtitle={t('unconnectedAccountAlertDescription')}
+      subtitle={t('unconnectedAccountAlertDescription', [selectedIdentity?.name || 'This account', origin || 'this site' ])}
```

Comment:
Can we move "This account" and "this site" into the `en/messages.json`?

Reply:
does messages.json support default values when variables are missing/not provided?

Reply:
Sorry, I meant that we can move just the default into the messages. Something like:

```js
subtitle={t('unconnectedAccountAlertDescription', [selectedIdentity?.name || t('thisAccount'), origin || t('thisSite') ])}
```

Reply:
So that we aren't injecting English into a non-English string

Reply:
ahhh, yeah good call.